token/services/identity/deserializer/typed.go

```diff
@@ -23,7 +23,7 @@ var logger = logging.MustGetLogger("token-sdk.services.identity.deserializer")
 type TypedVerifierDeserializer interface {
 	DeserializeVerifier(typ string, raw []byte) (driver.Verifier, error)
 	Recipients(id driver.Identity, typ string, raw []byte) ([]driver.Identity, error)
-	GetOwnerAuditInfo(id driver.Identity, typ string, raw []byte, p driver.AuditInfoProvider) ([][]byte, error)
+	GetOwnerAuditInfo(id driver.Identity, typ string, raw []byte, p driver.AuditInfoProvider) ([]byte, error)
```

Comment:
We could use `identity.Type` for the `typ` input of these 3 methods.

Reply:
okay